License Usage Pointer Update (#3906)
chef/chef-server: docs-chef-io/content/server/license_usage.md

- The Chef Infra Server version.
- The domain name of the Chef Infra Server installation and the number of installations by collecting the following parameters:

>   - Reports the domain name associated with the most common number of logins registered with Infra Server. Examples are `progress.com`, `my-charity.org`, and `a-service-provider.net`.

Reviewer: ```suggestion
  - Reports the domain name associated with the most common number of logins registered with Chef Infra Server. For example, `progress.com` or `example.com`.
```
